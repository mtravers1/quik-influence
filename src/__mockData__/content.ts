const data = {
  homepage: {
    banner: {
      resource: 'quik-influence',
      page: 'Home Page',
      type: 'banner',
      content: {
        sub_header: 'QUIK INFLUENCE THE CAPABLE CRM',
        header: 'Find your customer with Quik Influence',
        header_desc: 'new desc',
        image_url: 'url',
      },
    },
  },
};

export default data;
