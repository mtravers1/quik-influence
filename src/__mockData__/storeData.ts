const initialState: any = {
  auth: {
    user: { admin: { avatar: '', firstName: '', lastName: '' } },
    loading: false,
  },
  nav: null,
  campaigns: {
    campaigns: null,
    loading: true,
  },
};

export default initialState;
