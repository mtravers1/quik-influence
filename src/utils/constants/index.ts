export const CONTENT_URL = '/content';
export const Q_TOKEN = '_q_inf';
export const DEFAULT_PAGE_SIZE = '100';
export const ADMINS_ID = [2];

export const FILE_STATUS = {
  PENDING: 'pending',
  PROCESSING: 'processing',
  UPLOADING: 'uploading',
  PAUSED: 'paused',
  COMPLETED: 'completed',
  FAILED: 'failed',
  IDLE: 'idle',
};

export const headers: string[] = [
  'phone',
  'avatar',
  'email',
  'firstName',
  'lastName',
  'address',
  'city',
  'dateOfBirth',
  'instagramId',
  'socialMediaHandle',
  'state',
  'country',
  'postalCode',
  'gender',
  'address2',
  'address3',
  'facebookHandle',
  'tiktokHandle',
  'twitterHandle',
];

export const formDataRelatedToSpecificUser = [
  'updatedAt',
  'twitterHandle',
  'avatar',
  'phone',
  'address2',
  'email',
  'address1',
  'postalCode',
  'socialMediaHandle',
  'lastName',
  'firstName',
  'instagramId',
  'tiktokHandle',
  'address',
  'address3',
  'facebookHandle',
];

export const MARKETING_ADMIN = ['can_approve_or_reject_campaign'];
export const ADMIN_USER_AFFILIATE = ['can_get_joinable_campaigns'];
export const CAN_HANDLE_INFLUENCERS = ['can_manage_influencers'];

export const MARKET_PLACE_NEW_PRODUCTS = 'new';
export const MARKET_PLACE_POPULAR_PRODUCTS = 'popular';

export const CART_CLICK_NAME = 'cart';
