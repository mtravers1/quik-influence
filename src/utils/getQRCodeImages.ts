import QRCode, { QRCodeToDataURLOptions } from 'qrcode';
import { QrCodeRawData } from 'types';

const defaultOptions: Partial<QRCodeToDataURLOptions> = {
  errorCorrectionLevel: 'M',
  width: 256,
};

export default async function getQRImages(
  list: QrCodeRawData[],
  options?: QRCodeToDataURLOptions,
): Promise<string[]> {
  const qrCodeUrlPromises = list.map(({ text }) =>
    QRCode.toDataURL(text, { ...defaultOptions, ...options }),
  );
  const images = await Promise.all(qrCodeUrlPromises);
  return images;
}
