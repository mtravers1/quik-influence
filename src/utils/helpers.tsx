import axios from 'axios';

// const baseurl = 'http://localhost:2022';
const baseurl = process.env.NEXT_PUBLIC_BACKEND_URL;

import { DropdownSelectOption } from 'components/DropdownSelect';

const _axiosInstance = axios.create({
  baseURL: `${baseurl}/api/v1`,
  headers: {
    'Access-Control-Allow-Headers':
      'Access-Control-Allow-Origin, Access-Control-Allow-Headers, Content-Type',
    'Access-Control-Allow-Origin': '*',
  },
});

_axiosInstance.interceptors.request.use((config: any) => {
  let token: any = '';
  if (typeof window !== 'undefined') {
    token = localStorage.getItem('token');
  }
  config.headers.token = token;
  return config;
});

export const axiosInstance = _axiosInstance;

export const validate = (field: any, Regex: any, pattern: any) => {
  if (pattern.test(field)) return true;
  return false;
};

export const setToken = (token: string) => {
  axiosInstance.interceptors.request.use((config: any) => {
    config.headers.token = token ? token : '';
    return config;
  });
};

export const getNumberRange = (
  start: number,
  stop: number,
  step: number
): DropdownSelectOption[] =>
  Array.from({ length: (stop - start) / step + 1 }, (_, i) => ({
    label: (start + i * step).toString(),
    value: (start + i * step).toString(),
  }));
