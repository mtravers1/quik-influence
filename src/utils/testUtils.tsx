import { FC, ReactElement } from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import { applyMiddleware, createStore } from 'redux';
import thunk from 'redux-thunk';
import rootReducer from 'redux/reducers';

function makeTestStore(rootReducer: any, initialState: any) {
  const store = createStore(rootReducer, initialState, applyMiddleware(thunk));
  const origDispatch = store.dispatch;
  store.dispatch = jest.fn(origDispatch);
  return store;
}

const renderWithStore = (ui: ReactElement, storedefaults: any) => {
  const { initialState, ...renderOptions } = storedefaults;
  const store = makeTestStore(rootReducer, initialState);

  const Wrapper: FC = ({ children }) => {
    return <Provider store={store}>{children}</Provider>;
  };

  return render(ui, { wrapper: Wrapper, ...renderOptions });
};

export * from '@testing-library/react';
export { renderWithStore };
