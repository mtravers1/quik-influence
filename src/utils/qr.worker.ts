import { QRCodeToDataURLOptions } from 'qrcode';
import { QrCodeRawData } from 'types';

import getQRCodeImages from './getQRCodeImages';

export async function generateImages(
  values: QrCodeRawData[],
  options?: QRCodeToDataURLOptions,
): Promise<string[]> {
  const qrCodeImages = await getQRCodeImages(values, options);
  return qrCodeImages;
}
