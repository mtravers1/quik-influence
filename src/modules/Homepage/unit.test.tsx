import Home, { getStaticProps } from '../../pages';
import { nav, staticContentData } from '__mockData__/content';
import initialState from '__mockData__/storeData';
import { renderWithStore } from 'utils/testUtils';

describe('Home page Tests', () => {
  afterAll(() => {
    jest.resetAllMocks();
    jest.resetModules();
  });

  test('renders Home Page', () => {
    const { container } = renderWithStore(
      <Home nav={nav} pageContent={staticContentData.props.pageContent} />,
      { initialState }
    );
    expect(container).toMatchSnapshot();
  });

  test('getStaticProps should return data', async () => {
    const staticData = await getStaticProps({});

    expect(staticData).toEqual(staticContentData);
  });
});
