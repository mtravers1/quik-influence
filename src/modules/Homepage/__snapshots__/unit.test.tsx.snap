// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page Tests renders Home Page 1`] = `
<div>
  <main
    class="main"
  >
    <header
      class="css-k9u1u2"
    >
      <nav
        class="css-1dldq37"
      >
        <div
          class="css-1gpaoxy"
        >
          <div
            class="css-z2iays"
          >
            Get help for this site or any of our apps with - QUIK ASSISTANT
          </div>
          <div
            class="css-to8a61"
          >
            <a
              class="chakra-link css-g77k4c"
              href="/video-help"
            >
              <button
                class="chakra-button css-18s7f0k"
                type="button"
              >
                Get Video Help
              </button>
            </a>
            <a
              class="chakra-link css-0"
              href="/text-help"
            >
              <button
                class="chakra-button css-18s7f0k"
                type="button"
              >
                Get Text Help
              </button>
            </a>
          </div>
        </div>
        <div>
          <div
            class="css-eqo9nf"
          >
            <div
              class="css-ncghwh"
            >
              <img
                alt="logo"
                class="chakra-image css-1kr8y19"
                src="/logo-white.png"
              />
              <div
                class="css-1e9igf2"
              >
                <div
                  class="css-0"
                >
                  <a
                    class="chakra-link css-1km2gv8"
                    href="/"
                    style="color: red;"
                  >
                    Home
                  </a>
                </div>
              </div>
            </div>
          </div>
        </div>
      </nav>
    </header>
    <section
      class="css-1aw2lrc"
    >
      <div
        class="css-6m9kzq"
      />
      <div
        class="css-p7r2ak"
      >
        <div
          class="css-1fzwqmz"
        >
          <div
            class="css-1sufl1u"
          >
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <h3
                class="css-ux3rt2"
                contenteditable="false"
                id="banner_sub_header"
                tabindex="-1"
              >
                QUIK INFLUENCE THE CAPABLE CRM
              </h3>
            </div>
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <h1
                class="css-e30yqx"
                contenteditable="false"
                id="banner_header"
                tabindex="-1"
              >
                Find your customer with Quik Influence
              </h1>
            </div>
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <p
                class="css-kwm4yt"
                contenteditable="false"
                id="banner_header_desc"
                tabindex="-1"
              >
                QuikInfluence is a new cutting-edge media publisher. With its foundation built on partnerships with many of the largest influencers on social media and a network of over a dozen proprietary apps, our data feeds and ad placement options are not found anywhere else. Add in some revolutionary technology collaborations and it’s everything you’ve been waiting for in online marketing! Official launch scheduled for late March 2022, stay turned for more details. Can’t wait until the launch? Please contact our partners at Email Agency for more info on our platform at 561-899-0712 or Amie@emailagency.com
              </p>
            </div>
          </div>
          <div
            class="css-fvtyyp"
          >
            <img
              alt="Homepage"
              class="chakra-image css-bh4wo8"
              src="/homepage.jpg"
            />
          </div>
        </div>
        <form
          class="css-1x7pkf7"
        >
          <div
            class="css-126pau8"
          >
            <div
              class="chakra-form-control css-1ovjf9q"
              role="group"
            >
              <div
                class="css-79elbk"
              >
                 
                <div
                  class="css-79elbk"
                >
                  <input
                    aria-required="true"
                    class="chakra-input css-ip3tq6"
                    datatest-id="test-name"
                    id="field-1"
                    name="name"
                    placeholder="Your Name"
                    required=""
                    value=""
                  />
                </div>
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-user "
                  color="red"
                  data-icon="user"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
                  viewBox="0 0 448 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              
            </div>
            <div
              class="chakra-form-control css-1ovjf9q"
              role="group"
            >
              <div
                class="css-79elbk"
              >
                 
                <div
                  class="css-79elbk"
                >
                  <input
                    aria-required="true"
                    class="chakra-input css-ip3tq6"
                    datatest-id="test-phone"
                    id="field-2"
                    name="phone"
                    placeholder="Phone Number"
                    required=""
                    value=""
                  />
                </div>
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-phone "
                  color="red"
                  data-icon="phone"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M493.4 24.6l-104-24c-11.3-2.6-22.9 3.3-27.5 13.9l-48 112c-4.2 9.8-1.4 21.3 6.9 28l60.6 49.6c-36 76.7-98.9 140.5-177.2 177.2l-49.6-60.6c-6.8-8.3-18.2-11.1-28-6.9l-112 48C3.9 366.5-2 378.1.6 389.4l24 104C27.1 504.2 36.7 512 48 512c256.1 0 464-207.5 464-464 0-11.2-7.7-20.9-18.6-23.4z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              
            </div>
            <div
              class="chakra-form-control css-1ovjf9q"
              role="group"
            >
              <div
                class="css-79elbk"
              >
                 
                <div
                  class="css-79elbk"
                >
                  <input
                    aria-required="true"
                    class="chakra-input css-ip3tq6"
                    datatest-id="test-email"
                    id="field-3"
                    name="email"
                    placeholder="Your Email Address"
                    required=""
                    value=""
                  />
                </div>
                <svg
                  aria-hidden="true"
                  class="svg-inline--fa fa-mail-bulk "
                  color="red"
                  data-icon="mail-bulk"
                  data-prefix="fas"
                  focusable="false"
                  role="img"
                  style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
                  viewBox="0 0 576 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M160 448c-25.6 0-51.2-22.4-64-32-64-44.8-83.2-60.8-96-70.4V480c0 17.67 14.33 32 32 32h256c17.67 0 32-14.33 32-32V345.6c-12.8 9.6-32 25.6-96 70.4-12.8 9.6-38.4 32-64 32zm128-192H32c-17.67 0-32 14.33-32 32v16c25.6 19.2 22.4 19.2 115.2 86.4 9.6 6.4 28.8 25.6 44.8 25.6s35.2-19.2 44.8-22.4c92.8-67.2 89.6-67.2 115.2-86.4V288c0-17.67-14.33-32-32-32zm256-96H224c-17.67 0-32 14.33-32 32v32h96c33.21 0 60.59 25.42 63.71 57.82l.29-.22V416h192c17.67 0 32-14.33 32-32V192c0-17.67-14.33-32-32-32zm-32 128h-64v-64h64v64zm-352-96c0-35.29 28.71-64 64-64h224V32c0-17.67-14.33-32-32-32H96C78.33 0 64 14.33 64 32v192h96v-32z"
                    fill="currentColor"
                  />
                </svg>
              </div>
              
            </div>
          </div>
          <button
            class="chakra-button css-ht105s"
            type="button"
          >
            CONTACT US
          </button>
        </form>
      </div>
    </section>
    <section
      class="css-15a0p7b"
    >
      <div
        class="css-p7r2ak"
      >
        <div
          class="css-79elbk"
        >
          <div
            class="css-g90ssd"
          >
            <img
              alt="Close"
              src="/close.png"
              style="width: 10px; height: 10px;"
            />
          </div>
          <h2
            class="css-1tpxf1l"
            contenteditable="false"
            id="info_header"
            tabindex="-1"
          >
            Our Specialized apps and services make customer acquisition easy
          </h2>
        </div>
        <div
          class="css-79elbk"
        >
          <div
            class="css-g90ssd"
          >
            <img
              alt="Close"
              src="/close.png"
              style="width: 10px; height: 10px;"
            />
          </div>
          <p
            class="css-1h356hx"
            contenteditable="false"
            id="info_sub_header"
            tabindex="-1"
          >
            We craft unique digital experiences, Because of this we have an endless supply of leads all organic and verified.
          </p>
        </div>
        <div
          class="css-b69u32"
        >
          <div
            class="css-rxw4r2"
          >
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <h3
                class="css-1mws978"
                contenteditable="false"
                id="info_cards__0__header"
                tabindex="-1"
              >
                Industry Experts
              </h3>
            </div>
            <img
              class="chakra-image css-20teaf"
              src="/icon.png"
            />
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <p
                class="css-1yxu09j"
                contenteditable="false"
                id="info_cards__0__desc"
                tabindex="-1"
              >
                Not sure what your looking for? Our team can help you dial in on what will be your next audience.
              </p>
            </div>
          </div>
          <div
            class="css-qvkken"
          >
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <h3
                class="css-14oezjm"
                contenteditable="false"
                id="info_cards__1__header"
                tabindex="-1"
              >
                100's of industries
              </h3>
            </div>
            <img
              class="chakra-image css-20teaf"
              src="/icon.png"
            />
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <p
                class="css-16qv2i2"
                contenteditable="false"
                id="info_cards__1__desc"
                tabindex="-1"
              >
                Our leads are sourced from a Multi-Channel system that spans across many different platforms and industries
              </p>
            </div>
          </div>
          <div
            class="css-rxw4r2"
          >
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <h3
                class="css-1mws978"
                contenteditable="false"
                id="info_cards__2__header"
                tabindex="-1"
              >
                Custom experience
              </h3>
            </div>
            <img
              class="chakra-image css-20teaf"
              src="/icon.png"
            />
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <p
                class="css-1yxu09j"
                contenteditable="false"
                id="info_cards__2__desc"
                tabindex="-1"
              >
                Our amazing development team can create an expereince tailored to your comapnies needs. No matter what.
              </p>
            </div>
          </div>
        </div>
        <div
          class="css-79elbk"
        >
          <div
            class="css-g90ssd"
          >
            <img
              alt="Close"
              src="/close.png"
              style="width: 10px; height: 10px;"
            />
          </div>
          <p
            class="css-crqql1"
            contenteditable="false"
            id="info_footer"
            tabindex="-1"
          >
            Contact us about custom plans and agreements to ensure we meet your specific needs!
          </p>
        </div>
      </div>
    </section>
    <section
      class="css-1o6wpkk"
    >
      <div
        class="css-m5rldb"
      >
        <div
          class="css-1x741jx"
        >
          <div
            class="css-153k912"
          >
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <h3
                class="css-1a96edy"
                contenteditable="false"
                id="footer_sub_header"
                tabindex="-1"
              >
                Connecting dots one lead at a time.
              </h3>
            </div>
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <h1
                class="css-1lx5veh"
                contenteditable="false"
                id="footer_header"
                tabindex="-1"
              >
                Ready to find your next audience with us?
              </h1>
            </div>
            <div
              class="css-79elbk"
            >
              <div
                class="css-g90ssd"
              >
                <img
                  alt="Close"
                  src="/close.png"
                  style="width: 10px; height: 10px;"
                />
              </div>
              <p
                class="css-1dombsd"
                contenteditable="false"
                id="footer_header_desc"
                tabindex="-1"
              >
                No matter the size of the project, Quik Influence can create a custom service plan for you! And with access to leads in a seeminly limitless field of industries and niches, it’s no wonder we’re the go to for CRM needs.
              </p>
            </div>
          </div>
          <form
            class="css-1iq1sz"
          >
            <div
              class="css-12r6igv"
            >
              <div
                class="chakra-form-control css-15ucb8w"
                role="group"
              >
                <div
                  class="css-79elbk"
                >
                  <div
                    class="css-79elbk"
                  >
                    <input
                      aria-required="true"
                      class="chakra-input css-1h61lp8"
                      datatest-id="test-phone"
                      id="field-4"
                      name="phone"
                      placeholder="Phone Number"
                      required=""
                      value=""
                    />
                  </div>
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-phone "
                    color="red"
                    data-icon="phone"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
                    viewBox="0 0 512 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M493.4 24.6l-104-24c-11.3-2.6-22.9 3.3-27.5 13.9l-48 112c-4.2 9.8-1.4 21.3 6.9 28l60.6 49.6c-36 76.7-98.9 140.5-177.2 177.2l-49.6-60.6c-6.8-8.3-18.2-11.1-28-6.9l-112 48C3.9 366.5-2 378.1.6 389.4l24 104C27.1 504.2 36.7 512 48 512c256.1 0 464-207.5 464-464 0-11.2-7.7-20.9-18.6-23.4z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
                
              </div>
              <div
                class="chakra-form-control css-15ucb8w"
                role="group"
              >
                <div
                  class="css-79elbk"
                >
                  <div
                    class="css-79elbk"
                  >
                    <input
                      class="chakra-input css-1h61lp8"
                      datatest-id="test-isInfluencer"
                      id="field-5"
                      name="isInfluencer"
                      placeholder="Are You an Influencer"
                      value=""
                    />
                  </div>
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-phone "
                    color="red"
                    data-icon="phone"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
                    viewBox="0 0 512 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M493.4 24.6l-104-24c-11.3-2.6-22.9 3.3-27.5 13.9l-48 112c-4.2 9.8-1.4 21.3 6.9 28l60.6 49.6c-36 76.7-98.9 140.5-177.2 177.2l-49.6-60.6c-6.8-8.3-18.2-11.1-28-6.9l-112 48C3.9 366.5-2 378.1.6 389.4l24 104C27.1 504.2 36.7 512 48 512c256.1 0 464-207.5 464-464 0-11.2-7.7-20.9-18.6-23.4z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
                
              </div>
              <div
                class="chakra-form-control css-15ucb8w"
                role="group"
              >
                <div
                  class="css-79elbk"
                >
                  <div
                    class="css-79elbk"
                  >
                    <input
                      aria-required="true"
                      class="chakra-input css-1h61lp8"
                      datatest-id="test-name"
                      id="field-6"
                      name="name"
                      placeholder="Your Name"
                      required=""
                      value=""
                    />
                  </div>
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-user "
                    color="red"
                    data-icon="user"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
                
              </div>
              <div
                class="chakra-form-control css-15ucb8w"
                role="group"
              >
                <div
                  class="css-79elbk"
                >
                  <div
                    class="css-79elbk"
                  >
                    <input
                      aria-required="true"
                      class="chakra-input css-1h61lp8"
                      datatest-id="test-company"
                      id="field-7"
                      name="company"
                      placeholder="Company Name"
                      required=""
                      value=""
                    />
                  </div>
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-user "
                    color="red"
                    data-icon="user"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
                    viewBox="0 0 448 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M224 256c70.7 0 128-57.3 128-128S294.7 0 224 0 96 57.3 96 128s57.3 128 128 128zm89.6 32h-16.7c-22.2 10.2-46.9 16-72.9 16s-50.6-5.8-72.9-16h-16.7C60.2 288 0 348.2 0 422.4V464c0 26.5 21.5 48 48 48h352c26.5 0 48-21.5 48-48v-41.6c0-74.2-60.2-134.4-134.4-134.4z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
                
              </div>
              <div
                class="chakra-form-control css-15ucb8w"
                role="group"
              >
                <div
                  class="css-79elbk"
                >
                  <div
                    class="css-79elbk"
                  >
                    <input
                      aria-required="true"
                      class="chakra-input css-1h61lp8"
                      datatest-id="test-email"
                      id="field-8"
                      name="email"
                      placeholder="Your Email Address"
                      required=""
                      value=""
                    />
                  </div>
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-mail-bulk "
                    color="red"
                    data-icon="mail-bulk"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
                    viewBox="0 0 576 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M160 448c-25.6 0-51.2-22.4-64-32-64-44.8-83.2-60.8-96-70.4V480c0 17.67 14.33 32 32 32h256c17.67 0 32-14.33 32-32V345.6c-12.8 9.6-32 25.6-96 70.4-12.8 9.6-38.4 32-64 32zm128-192H32c-17.67 0-32 14.33-32 32v16c25.6 19.2 22.4 19.2 115.2 86.4 9.6 6.4 28.8 25.6 44.8 25.6s35.2-19.2 44.8-22.4c92.8-67.2 89.6-67.2 115.2-86.4V288c0-17.67-14.33-32-32-32zm256-96H224c-17.67 0-32 14.33-32 32v32h96c33.21 0 60.59 25.42 63.71 57.82l.29-.22V416h192c17.67 0 32-14.33 32-32V192c0-17.67-14.33-32-32-32zm-32 128h-64v-64h64v64zm-352-96c0-35.29 28.71-64 64-64h224V32c0-17.67-14.33-32-32-32H96C78.33 0 64 14.33 64 32v192h96v-32z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
                
              </div>
              <div
                class="chakra-form-control css-15ucb8w"
                role="group"
              >
                <div
                  class="css-79elbk"
                >
                  <div
                    class="css-79elbk"
                  >
                    <input
                      class="chakra-input css-1h61lp8"
                      datatest-id="test-industry"
                      id="field-9"
                      name="industry"
                      placeholder="Industry"
                      value=""
                    />
                  </div>
                  <svg
                    aria-hidden="true"
                    class="svg-inline--fa fa-mail-bulk "
                    color="red"
                    data-icon="mail-bulk"
                    data-prefix="fas"
                    focusable="false"
                    role="img"
                    style="width: 10px; position: absolute; top: 50%; left: 30px; transform: translateY(-50%); z-index: 1;"
                    viewBox="0 0 576 512"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M160 448c-25.6 0-51.2-22.4-64-32-64-44.8-83.2-60.8-96-70.4V480c0 17.67 14.33 32 32 32h256c17.67 0 32-14.33 32-32V345.6c-12.8 9.6-32 25.6-96 70.4-12.8 9.6-38.4 32-64 32zm128-192H32c-17.67 0-32 14.33-32 32v16c25.6 19.2 22.4 19.2 115.2 86.4 9.6 6.4 28.8 25.6 44.8 25.6s35.2-19.2 44.8-22.4c92.8-67.2 89.6-67.2 115.2-86.4V288c0-17.67-14.33-32-32-32zm256-96H224c-17.67 0-32 14.33-32 32v32h96c33.21 0 60.59 25.42 63.71 57.82l.29-.22V416h192c17.67 0 32-14.33 32-32V192c0-17.67-14.33-32-32-32zm-32 128h-64v-64h64v64zm-352-96c0-35.29 28.71-64 64-64h224V32c0-17.67-14.33-32-32-32H96C78.33 0 64 14.33 64 32v192h96v-32z"
                      fill="currentColor"
                    />
                  </svg>
                </div>
                
              </div>
            </div>
            <button
              class="chakra-button css-17mu9ok"
              type="button"
            >
              CONTACT US
            </button>
          </form>
        </div>
      </div>
    </section>
  </main>
</div>
`;
